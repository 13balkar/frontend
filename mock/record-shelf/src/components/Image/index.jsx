import * as React from 'react';
import PropTypes from 'prop-types';
import './image.css';
const Image = ({ src }) => {
  return (
      <img className='img' src={src} alt="img" />
  );
};
Image.propTypes = {
  src: PropTypes.string.isRequired
};

export default Image;
