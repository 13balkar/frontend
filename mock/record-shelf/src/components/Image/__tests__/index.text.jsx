import { render } from '@testing-library/react';
import * as React from 'react';
import Image from '..';

describe('Image', () => {
  it('should render Image correctly', () => {
    const { asFragment } = render(<Image src='imgsrc' />);
    expect(asFragment()).toMatchSnapshot();
  });
});
