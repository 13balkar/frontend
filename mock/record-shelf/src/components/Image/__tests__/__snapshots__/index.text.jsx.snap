// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image should render Image correctly 1`] = `
<DocumentFragment>
  <img
    alt="img"
    class="img"
    src="imgsrc"
  />
</DocumentFragment>
`;
